cutils.h

```diff
@@ -586,7 +586,7 @@ static inline size_t js__malloc_usable_size(const void *ptr)
     return malloc_size(ptr);
 #elif defined(_WIN32)
     return _msize((void *)ptr);
-#elif defined(__linux__) || defined(__ANDROID__) || defined(__CYGWIN__) || defined(__FreeBSD__)
+#elif defined(__linux__) || defined(__ANDROID__) || defined(__CYGWIN__) || defined(__FreeBSD__) || defined(__GLIBC__)
```

Comment:
Musl does implement `malloc_usable_size` but doesn't define `__GLIBC__`

Reply:
Then the addition of `|| defined(__GLIBC__)` (note: OR) will not change how this `#elif` chain is evaluated on musl.

Reply:
Yeah sorry I read to quickly, you're right!